### PR TITLE
DELTA-1313 add new bindings endpoints

### PR DESCRIPTION
**Pull request by MihailTymoshenko**

### CODE REVIEW

**Review by dsavchenko-sm — APPROVED** [review]

**Comment by vbarhatov** [comment]
image

I expect o see only 3 public endpoints. No needs to have 4th that works with a single binding. 
And a common naming format is a good practice

File: spec/openapi.yaml [hunk 1]
```diff
@@ -11729,6 +11720,57 @@ components:
...
+              type: array
+            contentFileUri:
+              description: Content file name to get bindings for.
+              type: string
+            contextUid:
+              description: ContextUID to get bindings for.
+              format: uuid
+              type: string
+            bindingUids:
+              description: Unique binding identifier to delete.
```
Comment (vbarhatov) [inline spec/openapi.yaml:11741]: image

File: spec/openapi.yaml [hunk 2]
```diff
@@ -3453,6 +3325,125 @@ paths:
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
 
+  /context-api/v2/projects/{projectId}/bindings/list:
+    post:
+      summary: Get bindings
+      description: |-
+        Return bindings for specified parameters.
```
Comment (vbarhatov) [inline spec/openapi.yaml:3332]: `Returns a paginated list of bindings according to specified parameters.`

Comment (vbarhatov) [reply spec/openapi.yaml:3332]: There is no info about page size

File: spec/openapi.yaml [hunk 3]
```diff
@@ -11729,6 +11720,57 @@ components:
...
+            contextUid:
+              description: ContextUID to get bindings for.
+              format: uuid
+              type: string
+            bindingUids:
+              description: Unique binding identifier to delete.
+              items:
+                type: string
+              type: array
+          required:
```
Comment (vbarhatov) [inline spec/openapi.yaml:11745]: image
our validation works differently

**Review by vbarhatov — CHANGES_REQUESTED** [review]

File: spec/openapi.yaml (hunk 1 above)
Comment (MihailTymoshenko) [reply spec/openapi.yaml:11701]: fixed

File: spec/openapi.yaml (hunk 2 above)
Comment (MihailTymoshenko) [reply spec/openapi.yaml]: fixed

File: spec/openapi.yaml (hunk 3 above)
Comment (MihailTymoshenko) [reply spec/openapi.yaml]: fixed

**Review by vbarhatov — APPROVED** [review]

**Review by dsavchenko-sm — COMMENTED** [review]
👍

**Pull request merged**